src/lib/actions/click-outside.ts

```diff
@@ -0,0 +1,16 @@
+/** Dispatch event on click outside of node */
+export function clickOutside(node: any) {
+  const handleClick = (event: MouseEvent) => {
+    if (node && !node.contains(event.target) && !event.defaultPrevented) {
```

Comment:
Here we can assume this value is a `Node`.

```suggestion
    if (node && !node.contains(event.target as Node) && !event.defaultPrevented) {
```